src/community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/AbstractQueryResult.java

```diff
@@ -45,4 +47,12 @@ public Map<String, RootBuilder> getTemplateMap() {
     public void setTemplateMap(Map<String, RootBuilder> templateMap) {
         this.templateMap = templateMap;
     }
+
+    /**
+     * Indicates whether there is a next page of results, or <code>null</code> if unknown (check #numberMatched in that
```

Comment:
not sure to have understood correctly. Will that return false if unknown or will it return null?
In the latter case, this getter is using boolean primitiveType so it may throw an NPE.